server/src/internalClusterTest/java/org/elasticsearch/action/termvectors/GetTermVectorsIT.java

```diff
@@ -1040,19 +1038,6 @@ public void testArtificialDocWithPreference() throws InterruptedException, IOExc
         assertEquals("expected to find term statistics in exactly one shard!", 2, sumDocFreq);
     }
 
-    public void testTermVectorsWithIgnoredField() throws IOException, InterruptedException {
-        // setup indices
-        assertAcked(prepareCreate("index").setMapping("field", "type=long,ignore_malformed=true"));
-        ensureGreen();
-
-        // add a doc with a bad long field
-        indexRandom(true, prepareIndex("index").setId("1").setSource("{\"field\":\"foo\"}", XContentType.JSON));
-
-        // do a tv request for all fields, _ignored should be returned
-        TermVectorsResponse resp = client().prepareTermVectors("index", "1").setSelectedFields("*").get();
-        assertThat(resp.getFields().terms("_ignored").size(), greaterThan(0L));
-    }
-
```

Comment:
I think that given the changes you made to get, this can be restored. It should work again.